lib/web/app/transport.go

```diff
@@ -287,8 +287,10 @@ func (t *transport) resignAzureJWTCookie(r *http.Request) error {
 
 	// Create a new jwt key using the client public key to verify and parse the token.
 	clientJWTKey, err := jwt.New(&jwt.Config{
-		Clock:       t.c.clock,
-		PublicKey:   r.TLS.PeerCertificates[0].PublicKey,
+		Clock:     t.c.clock,
+		PublicKey: r.TLS.PeerCertificates[0].PublicKey,
+		// TODO(gabrielcorado): use the cluster name. This value must match the
+		// one used by the proxy.
```

Comment:
i doubt we will ever fix this. if we do, how do we plan to preserve backwards compatibility? is this jwt verification even necessary?

Reply:
I haven't looked into too many details on this, either. I just updated the comment because now it is confusing why we use the MSI endpoint here, so I wanted to clarify that it is just a placeholder.

We can leave here and look later. If the JWT verification is not required, we can also simplify this code.